graspologic/embed/mase.py

```diff
@@ -134,13 +135,15 @@ def _reduce_dim(self, graphs):
 
         # embed individual graphs
         embeddings = [
-            selectSVD(
-                graph,
-                n_components=n_components,
-                algorithm=self.algorithm,
-                n_iter=self.n_iter,
+            delayed(
```

Comment:
This should be `Parallel(n_jobs=self.n_jobs)(delayed(selectSVD)(args)`. There also no need for this to be in a list since `Parallel` returns a list.

You also need to make this `n_jobs` arg a parameter for MASE initialization

Reply:
Then we should have a new argument in class mase as n_jobs then only

Reply:
> Then we should have a new argument in class mase as n_jobs then only

yes, a keyword argument, like in e.g. [KMeans](https://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html)

Reply:
@loftusa  my one other concern is if we add a new argument to MASE then  we  also need to change all the other files which import it.

Reply:
yah, by default K-means just uses all processing cores. I don't think we'd need to change any other files which import MASE if we added a new keyword argument, right?

Reply:
Yess tht would be perfect then

Reply:
i am unable to think what to put in the init function of mase class like n_jobs=?.@can you point this out @loftusa